README.adoc

```diff
@@ -1,155 +1,173 @@
-== Stream applications
+== Stream Applications
 
-This repository provides a collection of components that help with various data integration use cases.
-It provides standalone Java functions that can be reused in end-user applications.
+In this repository, you will find a collection of components that can meet various data integration use cases and requirements.
 
-In addition to providing these functions, this repository is also used to generate out of the box Spring Cloud Stream applications those using the functions as a baseline.
-These applications can be run standalone as Spring Cloud Stream applications or as part of a data flow such as the one orchestrated using Spring Cloud Data Flow.
+The repository's primary focus is to provide a set of standalone Java functions that can be useful in the end-user
+applications as-is.
 
-The repository is divided into two sections - `Functions` and `Applications`. The former is used for hosting the various functions and the latter is for generating the apps.
+Besides, this repository builds on the Java functions to generate standalone Spring Cloud Stream applications that can run
+against Spring Cloud Stream's RabbitMQ or Apache Kafka binder implementations. It is also possible to extend the generator
```

Comment:
I'm not sure what rules we follow in this project, but best practice is to have a `One sentence per line`: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line.

Just my fine fingers though...

Reply:
That is up to the project owners, so it is beyond my realm. I try to match whatever the project does.